.env-devel

```diff
@@ -66,6 +66,7 @@ DIRECTOR_REGISTRY_CACHING=True
 
 EFS_DNS_NAME=fs-xxx.efs.us-east-1.amazonaws.com
 EFS_MOUNTED_PATH=/tmp/efs
+EFS_ENABLED_FOR_USERS=[]
```

Comment:
Key Reads like a boolean but value is a list, maybe name it `EFS_ONLY_ENABLED_FOR_USERIDS`

Reply:
and actually this is a `DEV` / `DEBUG` env-var so maybe prefix it accordingly I think simcore has a convention for this

Reply:
Changed to `EFS_ONLY_ENABLED_FOR_USERIDS` as proposed.